docs/UserGuide.md

```diff
@@ -10,60 +10,70 @@ While we were applying to internships in our second year, we quickly realised ho
 desired commands. A [command summary](#command-summary) is also available if you prefer to have a quick reference for
-our command syntax!
+our command formats!
+
+Throughout our user guide, you will encounter different text formats or call-out boxes. Here are what they indicate:<br>
+[Highlighted text](#): Links to jump to the corresponding sections in the user guide
+`Monospaced text`: Command formats to guide your usage of SOCareers
+<div markdown="span" class="alert alert-warning">
+**:warning: Warning**<br>
+Information that is important to know to prevent unexpected or undefined behaviour from SOCareers.
+</div>
 
-Throughout the user guide, you will encounter these call-out boxes. Here are what they indicate:
 <div markdown="span" class="alert alert-info">
-**:information_source: Information**<br>
-Additional information to enrich your understanding.
+**:pencil: Parameter Information**<br>
+Details about parameters and their constraints to ensure SOCareers works as intended.
 </div>
 
 <div markdown="span" class="alert alert-primary">
 **:bulb: Tip**<br>
 Handy tips to enhance your experience with SOCareers.
 </div>
 
-<div markdown="span" class="alert alert-warning">
-**:warning: Warning**<br>
-Information that is important to know to prevent unexpected or undefined behaviour from SOCareers.
+<div markdown="span" class="alert alert-secondary">
+**:information_source: Additional Information**<br>
+Additional information to enrich your understanding of SOCareers.
 </div>
 
 ## Table of Contents
 1. [Quick Start](#quick-start)
 1. [Features](#features)
    1. Adding a contact
-      1. [Adding a person to contacts](#adding-a-person-to-contacts--add-p)
-      2. [Adding a company to contacts](#adding-a-company-to-contacts--add-c)
-      3. [Adding an internship to a company contact](#adding-an-internship-to-a-company-contact--add-i)
+      1. [Adding a person to contacts](#adding-a-person-to-contacts-add-p)
+      2. [Adding a company to contacts](#adding-a-company-to-contacts-add-c)
+      3. [Adding an internship to a company contact](#adding-an-internship-to-a-company-contact-add-i)
    2. Deleting a contact
-      1. [Deleting a person from contacts](#deleting-a-person-from-contacts--delete-p)
-      2. [Deleting a company from contacts](#deleting-a-company-from-contacts--delete-c)
-      3. [Deleting an internship from a company contact](#deleting-an-internship-from-a-company-contact--delete-i)
+      1. [Deleting a person from contacts](#deleting-a-person-from-contacts-delete-p)
+      2. [Deleting a company from contacts](#deleting-a-company-from-contacts-delete-c)
+      3. [Deleting an internship from a company contact](#deleting-an-internship-from-a-company-contact-delete-i)
    3. Listing contacts
-      1. [Listing all persons](#listing-all-persons--list-p)
-      2. [Listing all companies](#listing-all-companies--list-c)
+      1. [Listing all persons](#listing-all-persons-list-p)
```

Comment:
I think that List commands should come before the delete commands, since we do use the list commands in the examples for delete

Reply:
Sounds good, will edit later